misc/enr/src/lib.rs

```diff
@@ -395,6 +400,11 @@ impl Enr {
         self.add_key("tcp6", tcp.to_be_bytes().to_vec(), keypair)
     }
 
+    /// Set attestation subnet bitfield.
+    pub fn set_attnets(&mut self, attnets: Vec<u8>, keypair: &Keypair) -> Result<bool, EnrError> {
```

Comment:
I think it would be nicer to have attnets to be of type `&[u8]` to be more versatile when updating.

Reply:
Makes sense. Will make the changes.